### ISSUE TITLE
Go failing to install

### ISSUE THREAD

**Issue by al1-ce** [lemoony/snipkit#157]
```
- go install github.com/lemoony/snipkit@latest
go: github.com/lemoony/snipkit@latest (in github.com/lemoony/snipkit@v1.4.0):
	The go.mod file for the module providing named packages contains one or
	more replace directives. It must not contain directives that would cause
	it to be interpreted differently than if it were the main module.
- go version
- go version go1.19.3 linux/amd64
```

**Comment by lemoony** [comment]
Yes, this is becaue snipkit relies on a fork of the `tview` library. I hoped that rivo/tview#677 would finally get merged but it doesn't look like it. 

I will resolve the tview fork and integrate the finder directive directly into snipkit soon which will solve this issue.

**Comment by lemoony** [comment]
Fixed with version 1.4.1
